data/account/participationRegistry.go

```diff
@@ -137,9 +141,17 @@ func (r ParticipationRecord) Duplicate() ParticipationRecord {
 		LastStateProof:    r.LastStateProof,
 		EffectiveFirst:    r.EffectiveFirst,
 		EffectiveLast:     r.EffectiveLast,
+		StateProofPK:      nil,
 		VRF:               &vrf,
 		Voting:            &voting,
 	}
+	if r.StateProofPK != nil {
```

Comment:
Move this logic to line 132 and refactor to stay consistent with the way `VRF` and `Voting` are copied.